client/src/app/fridge/fridge.component.scss

```diff
@@ -0,0 +1,53 @@
+.ingredient {    
+    border: 2px;
+    border-radius: 10px;
+    margin-top: 2.5%;
+    margin-bottom: 2.5%;
+    padding: 20px 10px 20px 10px;
+   }
+
+.availableIngredient {
```

Comment:
nit: follow HTML CSS naming convention.

Reply:
renamed .availableIngredient to .available-ingredient, that's more inline with the naming convention now right?